chia/wallet/nft_wallet/nft_off_chain.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from pathlib import Path
+from typing import Optional
+
+import requests
+
+from chia.types.blockchain_format.program import Program
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.util.default_root import DEFAULT_ROOT_PATH
+from chia.util.hash import std_hash
+from chia.wallet.nft_wallet.nft_info import NFTCoinInfo
+from chia.wallet.nft_wallet.uncurry_nft import UncurriedNFT
+
+PREFIX_HASH_LENGTH = 3
+CACHE_PATH_KEY = "nft_metadata_cache_path"
```

Comment:
Shouldn't we also update initial-config.yaml to include this with a default value? Otherwise users will have a hard time figuring out how to override the default path (without looking at this code).